### PR TITLE
Changes for build and run on windows

### PR DESCRIPTION
**Pull request by moraxm**

### CODE REVIEW

File: fiware/CMakeLists.txt [hunk 1]
```diff
@@ -66,23 +78,36 @@ set_target_properties(${PROJECT_NAME}
...
+    PUBLIC
+        ${CURLPP_LIBRARY}
+        ${CURL_LIBRARIES}
+        yaml-cpp
+    PRIVATE
+        soss::core)
+else(WIN32)
 target_link_libraries(${PROJECT_NAME}
     PRIVATE
+		yaml-cpp
```
Comment (Jaime-Burgos) [inline fiware/CMakeLists.txt]: I think this is unnecessary in LINUX

File: fiware/CMakeLists.txt [hunk 2]
```diff
@@ -66,23 +78,36 @@ set_target_properties(${PROJECT_NAME}
...
         soss::core
         curlpp
-        pthread
-)
+        pthread)
```
Comment (Jaime-Burgos) [inline fiware/CMakeLists.txt]: Same here

File: fiware/CMakeLists.txt [hunk 3]
```diff
@@ -19,14 +19,26 @@ cmake_minimum_required(VERSION 3.5.0)
 project(soss-fiware)
 
 find_package(soss-core REQUIRED)
+find_package(yaml-cpp REQUIRED)
+include(FindCURL)
+find_package(curl REQUIRED)
```
Comment (Jaime-Burgos) [inline fiware/CMakeLists.txt:25]: This three lines should have an if(WIN32). They output an error in LINUX

File: fiware/CMakeLists.txt [hunk 4]
```diff
@@ -19,14 +19,26 @@ cmake_minimum_required(VERSION 3.5.0)
...
 
+find_path(CURL_INCLUDE_DIRS NAMES "include/curl/curl.h")
+include_directories( 
+  "${CURL_INCLUDE_DIRS}"
+)
+
+ include_directories( 
+  "${CURLPP_DIR}/include"
+)
+
```
Comment (Jaime-Burgos) [inline fiware/CMakeLists.txt:45]: This part too

**Review by Jaime-Burgos — CHANGES_REQUESTED** [review]

File: fiware/CMakeLists.txt (hunk 4 above)
Comment (moraxm) [reply fiware/CMakeLists.txt:45]: Done

File: fiware/CMakeLists.txt (hunk 3 above)
Comment (moraxm) [reply fiware/CMakeLists.txt:25]: Done

File: fiware/CMakeLists.txt (hunk 2 above)
Comment (moraxm) [reply fiware/CMakeLists.txt]: pthread was before this commit

File: fiware/CMakeLists.txt (hunk 1 above)
Comment (moraxm) [reply fiware/CMakeLists.txt]: Done

**Review by Jaime-Burgos — APPROVED** [review]

**Pull request merged**